source/future.tex

```diff
@@ -98,20 +98,64 @@
 prefix \tcode{++} and \tcode{\dcr} expressions\iref{expr.pre.incr}
 of \tcode{volatile}-qualified arithmetic and pointer types are deprecated.
 
+\begin{example}
+\begin{codeblock}
+volatile int velociraptor;
+++velociraptor;                 // deprecated: is this a single memory operation, or two
```

Comment:
Please remove the attempt to give implicit rationale here. Just say that this case is deprecated, nothing more.

Reply:
Done.